pallets/automation-price/src/lib.rs

```diff
@@ -314,12 +324,10 @@ pub mod pallet {
 
 	#[pallet::error]
 	pub enum Error<T> {
-		/// The provided_id cannot be empty
 		EmptyProvidedId,
```

Comment:
This Error related to the providedId is the old clientId pre-2.0.0. Is it being used by automationPrice?